Backpack/src/main/java/net/skyscanner/backpack/button/BpkButton.kt

```diff
@@ -76,6 +75,7 @@ open class BpkButton(
       }
 
       setPadding(paddingHorizontal, paddingVertical, paddingHorizontal, paddingVertical)
+      applyStyle(type.createStyle(context))
```

Comment:
There was a bug with dynamic changing of button type -- the background were the same